daemon/src/maker.rs

```diff
@@ -336,6 +337,8 @@ async fn main() -> Result<()> {
         .launch()
         .await?;
 
+    std::mem::drop(conn);
```

Comment:
Perhaps an explicit `conn.close()` might be even better?

Another way of solving this would be to limit the scope of `conn`:

```rust
let cfds = {
	let mut conn = db.acquire().await?;

	load_all_cfds(&mut conn).await?
};
```

Reply:
Limiting scope sounds best for me, I didn't opt for it initially as the connection was used for 2 distinct purposes - housekeeping and cfds. then it dawned at me that I could have just acquired 2 separate conns and make the scope much tighter!